torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -66,6 +62,10 @@ Value* MatchIfBlocksOutputForValue(
     Value* orig_data,
     Block* outer_block,
     Value* origOutput) {
+  if (outer_block->owningNode()->kind() == prim::Loop)
+    return outer_block->owningNode()->outputs().at(
+        outer_block->owningNode()->outputs().size() - 1);
```

Comment:
why return the last output for loop?

Reply:
There's nothing to match for loops, because there's only one block.
Will add a comment for this.